docs/source/bias_reduction.ipynb

```diff
@@ -11,66 +11,86 @@
    "cell_type": "markdown",
```

Comment:
<p>in general if an anomaly bias is already removed, trying to remove it again shouldnt change anything, should it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/434/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>